sdk/metricsadvisor/ai-metrics-advisor/src/metricsAdvisorAdministrationClient.ts

```diff
@@ -43,7 +43,8 @@ import {
   AnomalyAlertConfiguration,
   DataSourceCredentialEntityUnion,
   DataSourceCredentialPatch,
-  CredentialsPageResponse
+  CredentialsPageResponse,
+  RestResponse
```

Comment:
@sarangan12 The `RestResponse` is imported here and used in all the delete APIs in this file